.github/workflows/release-python-client.yaml

```diff
@@ -0,0 +1,55 @@
+name: Publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+
+on: push
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: set semver
+        run: |
+          CURRENT_SEMVER=$(curl -s -X GET "https://jade.datarepo-dev.broadinstitute.org/configuration" -H "accept: application/json" | jq -r '.semVer|rtrimstr("-SNAPSHOT")')
+          echo "CURRENT_SEMVER=$(echo ${CURRENT_SEMVER})" >> $GITHUB_ENV
+      - name: Generate Python client with openapi-generator
+        # openapi-generator-cli generate -i src/main/resources/api/data-repository-openapi.yaml -g python -o ../data-repo-client --package-name data_repo_client
+        # -c /github/workspace/${{ inputs.config-file }} \
+        run: |
+          docker run -u 1001 --rm --workdir /github/workspace -v "$GITHUB_WORKSPACE":"/github/workspace" \
+          openapitools/openapi-generator-cli:v4.3.1 generate \
+          -i /github/workspace/src/main/resources/api/data-repository-openapi.yaml \
+          -g python \
+          -o /github/workspace/data-repo-client \
+          --additional-properties=projectName=data-repo-client,packageName=data_repo_client,packageVersion=${CURRENT_SEMVER} \
+          --skip-validate-spec
+      - name: Install pypa/build
+        working-directory: ./data-repo-client
+        run: >-
+          python -m
+          pip install
+          build
+          --user
+      - name: Build a binary wheel and a source tarball
+        working-directory: ./data-repo-client
+        run: >-
+          python -m
+          build
+          --sdist
+          --wheel
+          --outdir dist/
+          .
+      - name: Publish distribution 📦 to Test PyPI
+        uses: pypa/gh-action-pypi-publish@master
+        with:
+          password: ${{ secrets.TEST_PYPI_API_TOKEN }}
+          repository_url: https://test.pypi.org/legacy/
+          packages_dir: ./data-repo-client/dist
+#      - name: Publish distribution 📦 to PyPI
+#        if: startsWith(github.ref, 'refs/tags')
+#        uses: pypa/gh-action-pypi-publish@master
+#        with:
+#          password: ${{ secrets.PYPI_API_TOKEN }}
```

Comment:
We have a special way of handling not-trusted external libraries that we want to use for github action steps. I believe we fork the repo into the "broadinstitute" organization and then reference the forked step. 
* Example of forked repo: [setup-helmfile](https://github.com/broadinstitute/setup-helmfile)
* Example of use in one our actions: [perf test run](https://github.com/DataBiosphere/jade-data-repo/blob/30f20a691eb0a42bcd36d5720e23b6efefa0aa50/.github/workflows/test-runner-on-perf.yml#L107)